torch/onnx/symbolic_opset17.py

```diff
@@ -47,6 +47,14 @@ def layer_norm(
     # layer_norm normalizes on the last D dimensions,
     # where D is the size of normalized_shape
     axis = -len(normalized_shape)
+    scalar_type = _type_utils.JitScalarType.from_value(input, _type_utils.JitScalarType.FLOAT)
```

Comment:
Would assume FLOAT the correct value? what if this is executed under an Autocast region? couldn't the right type would be float16 or bfloat16?

Reply:
If the JitScalarType cannot be fetched from value, FLOAT will be used here. Maybe it's better not to use FLOAT as default? Just raise error if from_value returns None?

Reply:
We use `FLOAT` by default in other ops. I think we can give it a try the way it is